src/css/game-grid.css

```diff
@@ -1,4 +1,13 @@
 .grid-container {
+  display: flex;
+  justify-content: center;
+  /*grid-template-columns: repeat(16, 1fr);
+  grid-template-rows: 60px;*/
+
+}
+
```

Comment:
Is there a particular reason you made this change? I've been working on this since last night, I thought you were working on other sections.